components/SegmentedControl/SegmentedControl.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { Row, Col } from '../Grid';
+import { colors, spacing } from '../shared/theme';
+import SegmentedButton from './SegmentedButton';
+
+const SegmentedWrapper = styled.div`
+  display: flex;
+  justify-content: space-around;
+
+  label {
+    border: 1px solid;
+    border-right: 0px;
+    ${({
+      theme: {
+        colors: {
+          primary: { 700: primaryColor700 },
+        },
+        spacing: { xsmall, xxsmall },
+      },
+    }) => `
+      border-color: ${primaryColor700};
+      padding: ${xxsmall}px ${xsmall}px;
+    `}
+
+    &:first-child {
+      border-radius: 4px 0px 0px 4px;
+    }
+
+    &:last-child {
+      border-radius: 0px 4px 4px 0px;
+      ${({
+        theme: {
+          colors: {
+            primary: { 700: primaryColor700 },
+          },
+        },
+      }) => `
+        border: 1px solid ${primaryColor700};
+      `}
+    }
+  }
+`;
+
+const MAX_NUM_BUTTONS = 5;
+
+const SegmentedControl = ({ items, name, onChange, theme, darkMode }) => {
+  const ButtonDefaultChecked = allItems => {
+    const itemChecked = allItems.filter(item => item.checked);
+    if (typeof itemChecked !== 'undefined' && itemChecked.length > 0)
+      return itemChecked[0].value;
+
+    return undefined;
```

Comment:
you could use `[]` instead of `undefined`. It's more efficient than return `undefined` because you can't need to use `typeof === 'undefined'` to valid this checked. What do you think about that?

Reply:
Good one! Thanks!